app/frontend/wallet/max-amount-calculator.ts

```diff
@@ -24,15 +26,27 @@ export const MaxAmountCalculator = () => {
         profitableInputs.map(({tokenBundle}) => tokenBundle)
       )
       const inputBalance = getInputBalance(profitableInputs)
-      const additionalLovelaceAmount =
+
+      const changeOutputs = splitTxOutput(
+        {
+          isChange: false,
```

Comment:
probably doesn't matter here as it is just for the sake of a calculation, but it seems fishy the flag is false while the variable name is changeOutputs